### DISCUSSION TITLE
Auto tailing/refreshing logs

### DISCUSSION THREAD

**Discussion by lssatvik** [apache/airflow#26519] [Q&A] [answered]
I desire my task logs to automatically refresh, or stream for the latest try. It doesn't happen for me, and I noticed these configurations:-

![image](https://user-images.githubusercontent.com/3456652/191238758-aea25fbc-74e1-43e1-a545-a275aacb9efa.png)

I also see that this has been implemented in ti_log.js.

![image](https://user-images.githubusercontent.com/3456652/191238936-fabdda3b-7c21-4287-abb3-d26c2a07b814.png)

But not implemented (to my understanding) in FileTaskHandler:-

![image](https://user-images.githubusercontent.com/3456652/191239129-63349e0d-854d-473a-a0f0-b66a38802b8a.png)

May I know what this configuration/feature is? Whether its implemented or not, and possibly how to set up/implement this feature on my installation.

**Comment by potiuk** [comment, marked as answer]
I think this is coming in 2.4.1 - merged PR (yesterday) here: https://github.com/apache/airflow/pull/26169 - you can upgrade to 2.4.0 and apply the patch and see if it solves your problem.

**Reply by lssatvik** [reply]
Ported into 2.3.4, and it works. Thanks!

**Reply by potiuk** [reply]
Cool. Great that you tested it too :)
